src/frontend/src/scheduler/hummock_snapshot_manager.rs

```diff
@@ -122,6 +132,11 @@ impl HummockSnapshotManager {
         })
     }
 
+    pub fn update_epoch(&self, epoch: u64) {
+        println!("received epoch {}", epoch);
```

Comment:
remove this?

Reply:
I'm just debugging a failure in regression test.